src/app/components/item-data.ts

```diff
@@ -125,17 +125,41 @@ export class ItemQuestionsComponent implements OnDestroy {
     this.preparedQuestions = this.store.preparedQuestions;
     this.currentQuestion = this.store.currentQuestion;
     this.redashUrl = this.itemService.getRedashUrl(this.store.dataQuery);
+
+    // Create a filename - item name + current question, so for example:
+    // wingate institute__annual summary of communications
+
+    // For the name, take either name, title, page title
+    let entityName = '';
+    if (this.store.item.name) {
+      entityName = this.store.item.name;
+    } else if (this.store.item.title) {
+      entityName = this.store.item.title;
+    } else if (this.store.item.page_title) {
+      entityName = this.store.item.page_title;
+    }
+
+    // Create the question
+    let question = '';
+    for (let i = 0; i < this.store.currentQuestion.parsed.length; i++) {
+      question = question + this.store.currentQuestion.parsed[i].value;
+    }
+
+    const fileName = decodeURIComponent(entityName + '__' + question);
```

Comment:
`decodeURIComponent` makes sense when we're decoding something from a url - which is not the case here...

Reply:
fixed it